searx/plugins/twitter_rewrite.py

```diff
@@ -0,0 +1,38 @@
+'''
+searx is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+searx is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with searx. If not, see < http://www.gnu.org/licenses/ >.
+
+(C) 2015 by Adam Tauber, <asciimoo@gmail.com>
+'''
```

Comment:
It is your implementation, you hold the copyright, not Adam Tauber!

- Copyright is the legal term used to declare and prove who owns the intellectual property (the code, text, etc.).
- Licensing is the legal term used to describe the terms under which people are allowed to use the copyrighted material.

@asciimoo what do you think .. should we switch to [SPDX License Identifier](https://spdx.org/using-spdx-license-identifier)? .. see issue  #1801

Reply:
Yeah, SPDX is good for me.